cql3/Cql.g

```diff
@@ -1396,8 +1396,9 @@ columnOperation[operations_type& operations]
 
 columnOperationDifferentiator[operations_type& operations, ::shared_ptr<cql3::column_identifier::raw> key]
     : '=' normalColumnOperation[operations, key]
-    | '[' k=term ']' specializedColumnOperation[operations, key, k, false]
-    | '[' K_SCYLLA_TIMEUUID_LIST_INDEX '(' k=term ')' ']' specializedColumnOperation[operations, key, k, true]
+    | '[' k=term ']' collectionColumnOperation[operations, key, k, false]
+    | '.' field=ident udtColumnOperation[operations, key, field]
+    | '[' K_SCYLLA_TIMEUUID_LIST_INDEX '(' k=term ')' ']' collectionColumnOperation[operations, key, k, true]
     ;
 
```

Comment:
This should be the last patch, no?

Reply:
You mean the one which adds the feature of setting single fields?
Not necessarily. The deleter patch is independent from this patch and their relative order doesn't matter. Some fixes like the trailing nulls patch can also be after this one. Also there are patches that add tests, which I could merge with the other patches, but I was already too tired with all the rebasing to split the test patches in a way that would match the other patches.